packages/vega-schema/src/on-events.js

```diff
@@ -1,72 +1,49 @@
-export default {
-  "defs": {
-    "listener": {
-      "oneOf": [
-        {"$ref": "#/refs/signal"},
-        {
-          "type": "object",
-          "properties": {
-            "scale": {"type": "string"}
-          },
-          "required": ["scale"]
-        },
-        {"$ref": "#/defs/stream"}
-      ]
-    },
+import {
+  array, allOf, def, oneOf, object, ref,
+  anyType, booleanType, stringType, signalRef
+} from './util';
+
+// ???
```

Comment:
???

Reply:
'???' is an extremely detailed and informative comment to myself to revisit later to see if I want to consolidate reference values as a set of exports rather than recreate them across various schema generation files.